src/dotty/tools/dotc/config/ScalaSettings.scala

```diff
@@ -88,6 +88,7 @@ class ScalaSettings extends Settings.SettingGroup {
   val Xshowobj = StringSetting("-Xshow-object", "object", "Show internal representation of object.", "")
   val showPhases = BooleanSetting("-Xshow-phases", "Print a synopsis of compiler phases.")
   val sourceReader = StringSetting("-Xsource-reader", "classname", "Specify a custom method for reading source files.", "")
+  val XnoValueClasses = BooleanSetting("-Xno-value-classes", "Do not use value classes. Helps debugging.")
```

Comment:
I would actually used it during my everyday development and would recommend others, because it simplifies debugging.
Though I'm fine with it being either `-X` or `-Y`